rules-reviewed/eap7/eap6/hsearch.windup.xml

```diff
@@ -0,0 +1,899 @@
+                    <tag>hibernate-search</tag>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="hsearch-00114">
+            <when>
+                <javaclass references="org.hibernate.search.engine.service.spi.ServiceProvider">
+                    <location>IMPLEMENTS_TYPE</location>
+                </javaclass>
+            </when>
+            <perform>
+                <hint title="Implementing ServiceProvider in Hibernate Search 5" effort="1"
+                    severity="mandatory">
+                    <message>The `org.hibernate.search.engine.service.spi.ServiceProvider` SPI has been refactored.
+                    If you where integrating with the old service contract, refer to the javadoc of `ServiceManager`, `Service`, `Startable` and `Stoppable` for details about the new contract.</message>
+                    <link
+                        href="https://access.qa.redhat.com/documentation/en/red-hat-jboss-enterprise-application-platform/version-7.0/migration-guide/#migrate_miscellaneous_hibernate_search_changes"
+                        title="Miscelanous Changes in Hibernate Search 5.x" />
+                    <link
+                        href="http://docs.jboss.org/hibernate/search/5.0/api/org/hibernate/search/engine/service/spi/ServiceManager.html"
+                        title="Hibernate Search javadoc for ServiceManager" />
+                    <link href="http://docs.jboss.org/hibernate/search/5.0/api/org/hibernate/search/engine/service/spi/Service.html"
+                        title="Hibernate Search javadoc for Service" />
+                    <link
+                        href="http://docs.jboss.org/hibernate/search/5.0/api/org/hibernate/search/engine/service/spi/Stoppable.html"
+                        title="Hibernate Search javadoc for Stoppable" />
+                    <link
+                        href="http://docs.jboss.org/hibernate/search/5.0/api/org/hibernate/search/engine/service/spi/Startable.html"
+                        title="Hibernate Search javadoc for Startable" />
+                    <tag>hibernate-search</tag>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="hsearch-00115">
+            <when>
+                <javaclass references="org.hibernate.search.annotations.IndexedEmbedded">
+                    <location>ANNOTATION</location>
+                </javaclass>
+            </when>
+            <perform>
+                <hint title="Indexing of id Fields of Embedded Relations" effort="1" severity="mandatory">
+                    <message><![CDATA[
+                     When using an `@IndexedEmbedded` annotation to include fields from a related entity,
+                     the id of the related entity is no longer included.
+
+                     You can enable the inclusion of the id by using the `includeEmbeddedObjectId` attribute of the `@IndexedEmbedded` annotation.
+
+                    ```java
```

Comment:
This seems to be not supported in Windup. In the migration report, I see a code block beginning with "\`\`\`java" and a huge indentation.